src/resdk/tables/base.py

```diff
@@ -213,6 +223,30 @@ def _metadata_version(self) -> str:
         )
         return version
 
+    @property
+    @lru_cache()
+    def _qc_version(self) -> str:
+        """Return server QC data version.
+
+        The versioning of QC Data on the server is determined by the hash of
+        the tuple of sorted QC data objects ids.
+
+        :return: data version
+        """
+        mqc_ids = []
+        for mqc in self.collection.data.filter(
+            type="data:multiqc", status="OK", fields=["id", "entity__id"]
+        ):
+            if mqc.sample is None:
+                continue
+            mqc_ids.append(mqc.id)
+        if len(mqc_ids) == 0:
```

Comment:
It is probably more `Pythonic` to simplify the condition to  ``if mqs_ids:``

Reply:
Good point, thanks.